Sources/StreamChatUI/Utils/ImageCDN.swift

```diff
@@ -0,0 +1,74 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import UIKit
+
+public enum ImageCrop: String {
+    case top, bottom, left, right, center
+}
+
+public enum ImageResize: String {
+    case crop, scale, fill
+}
```

Comment:
These should not public since they're params for Stream CDN. Custom CDN's can have their own params

Reply:
It needs to be public because it's used in public protocol.

Reply:
Hmm, do you think we need to expose this to users? I'd be ok with hardcoding this to `fill + center` 🤔